### DISCUSSION TITLE
Update to support Statamic 3.4

### DISCUSSION THREAD

**Discussion by lloydowen** [duncanmcclean/simple-commerce#807] [Help] [answered]
I'm unable to update Statamic to 3.4 as simple commerce requires 3.3.*. It would be nice to have this package & its dependencies updated to allow this update 😄

**Comment by duncanmcclean** [comment, marked as answer]
Which version of Simple Commerce are you on?

You can use Statamic 3.4 from [v4.5.0](https://github.com/duncanmcclean/simple-commerce/releases/tag/v4.5.0) onwards.

**Reply by lloydowen** [reply]
Apologies, thank you.
